requirements.txt

```diff
@@ -1,26 +1,48 @@
+appdirs==1.4.4
 appnope==0.1.0
+attrs==19.3.0
 backcall==0.1.0
+certifi==2020.4.5.1
+chardet==3.0.4
 click==7.1.2
 decorator==4.4.2
+distlib==0.3.0
 dnspython==1.16.0
+filelock==3.0.12
 Flask==1.1.2
 Flask-PyMongo==2.3.0
+get==2019.4.13
 gunicorn==20.0.4
+idna==2.9
 ipdb==0.13.2
 ipython==7.14.0
 ipython-genutils==0.2.0
 itsdangerous==1.1.0
 jedi==0.17.0
 Jinja2==2.11.2
+jsonify==0.5
 MarkupSafe==1.1.1
+more-itertools==8.3.0
+packaging==20.4
 parso==0.7.0
 pexpect==4.8.0
 pickleshare==0.7.5
+pluggy==0.13.1
+post==2019.4.13
 prompt-toolkit==3.0.5
 ptyprocess==0.6.0
+public==2019.4.13
+py==1.8.1
 Pygments==2.6.1
 pymongo==3.10.1
+pyparsing==2.4.7
+pytest==5.4.2
+query-string==2019.4.13
+request==2019.4.13
+requests==2.23.0
```

Comment:
we should look into how we can clean up our dependencies in the future, just to avoid any unnecessary installations during the build process during deployment